lib/octokit/client/source_import.rb

```diff
@@ -9,9 +9,9 @@ module SourceImport
       # Start a source import to a GitHub repository using GitHub Importer.
       #
       # @param repo [Integer, String, Hash, Repository] A GitHub repository.
-      # @param vcs [String] The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
       # @param vcs_url [String] The URL of the originating repository.
       # @param options [Hash]
+      # @option options [String] :vcs The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
```

Comment:
> @LizzHale I think that's fine, this users know that is for a preview.

I agree with the point that users accept additional conditions when using preview APIs, but I would like to see that apply to the API itself only. 

In this case if someone is using the preview API raw, there code wont break because of it no longer being required, however that is not the case for this change in Octokit. Since wrappers are supposed to make things easier, I would like to see us gracefully deprecate this even for a preview API. I believe we should stick to our versioning for Octokit and not break APIs between nonmajor versions. We can promote the new API while supporting the existing deprecated one, then remove the deprecated call in the next major version.

I believe something along these lines would be fine? What do you all think?

```ruby
def start_source_import(repo, vcs_url, options = {}, _deprecated_args=nil)
  if _deprecated_args
    vcs = vcs_url
    vcs_url = options
    options = _deprecated_args.merge(:vcs => vcs)
  end

  options = ensure_api_media_type(:source_imports, options.merge(:vcs_url => vcs_url))
  put "#{Repository.path repo}/import", options
end
```